bin/metrics-docker-stats.rb

```diff
@@ -109,12 +115,24 @@ class DockerStatsMetrics < Sensu::Plugin::Metric::CLI::Graphite
          boolean: true,
          default: false
 
+  def create_container_list(container_list, pattern)
+    matched_list = []
+    container_list.each do |container|
+      if container =~ /#{pattern}/
+        matched_list.push(container)
+      end
+    end
+    matched_list
+  end
+
   def run
     @timestamp = Time.now.to_i
     @client = DockerApi.new(config[:docker_host])
 
     list = if config[:container] != ''
              [config[:container]]
+           elsif config[:container_list_regex] != ''
```

Comment:
this can alternatively be expressed as `elsif !config[:container_list_regex].empty?` but I think it might be best to remove the default empty string and check for `!config[:container_list_regex].nil?`